Gemfile

```diff
@@ -39,6 +39,11 @@ group :development, :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '~> 2.13'
   gem 'selenium-webdriver'
+
+  gem 'rspec-rails', '~> 3.7'
+  gem 'factory_bot_rails'
```

Comment:
Wow, cool 👍
Actually, I'm an old-fashioned guy loving `factory girl`.
Just out of curiosity, have you used the `factory_bot_rails ` before? And why it's better than factory girl?

Reply:
Yes, I've used it in this little project https://github.com/wazzuper/blogarium

All I know about difference between ```factory girl``` and ```factory_bot_rails``` - it's their names, this is the same gem, but with a different name :)